ci/test.ps1

```diff
@@ -0,0 +1,45 @@
+Set-StrictMode -Version Latest
```

Comment:
I think we should start adopting common prefixes based on the CI system. E.g. "travis-build.sh", "travis-test.sh", "appveyor-build.sh" etc. Right now our "ci/" dir is a mess. I don't see a need to adjust existing scripts in this PR, but it might make sense if newly added files started to already use such names.

Reply:
I totally agree. And in fact, I didn't mean to say that "ci/" is a mess, but "scripts/". I didn't realize at first that you've created the "ci/" directory in this PR, it already makes things a lot cleaner. I'm still not particular happy about just distinguishing scripts by their file ending, only, but if we're able to merge the new Unix CI-scripts and the old Travis CI-scripts in the short run I'm rather indifferent.